KnowWhereGraph/kwg-faceted-search
# Remove the about and explore pages

This PR removes the now obsoleted Explore and About components.

To test
1. Check this branch out
2. Make sure that the KE compiles and runs when running `ng serve`
